packages/strapi-plugin-upload/config/functions/bootstrap.js

```diff
@@ -52,3 +56,23 @@ const baseProvider = {
     throw new Error('Provider delete method is not implemented');
   },
 };
+
+const pruneObsoleteRelations = async name => {
+  const { orm } = strapi.plugins[name].models.file;
+
+  if (orm !== 'mongoose') {
+    return;
+  }
+
+  await strapi.query('file', 'upload').custom(pruneObsoleteRelationsQuery)();
```

Comment:
Should it be `pluginName` instead of `'upload'` ?
I'm thinking, do we need the name as a parameter ? Since it's only for this plugin

Reply:
I agree I don't think we need this variable. this would reduce readability.

Reply:
Makes sense, removed in 8a2050add05d12bdb9ed68e380bde1ec9c24ca75